pypeit/reduce.py

```diff
@@ -1485,7 +1485,8 @@ def apply_relative_scale(self, scaleImg):
         # Correct the relative illumination of the science frame
         msgs.info("Correcting science frame for relative illumination")
         self.scaleimg *= scaleImg.copy()
-        sciImg, varImg = flat.flatfield(self.sciImg.image.copy(), scaleImg.copy(), self.sciImg.fullmask,
+        sciImg, varImg = flat.flatfield(self.sciImg.image.copy(), scaleImg.copy(),
```

Comment:
I think numpy produces new images if you multiply an image by a number or array so the copy may not be necessary.

Reply:
Yep.  I removed these copies along with other changes to this part of the code.